laos_gggi/emdat_processing.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+import os
+from os.path import exists
+import const_vars
+
+PROB_COLS = const_vars.PROB_COLS
+INTENSITY_COLS = const_vars.INTENSITY_COLS
+DISASTERS_FOUND = const_vars.DISASTERS_FOUND
```

Comment:
you can just write `from const_vars import PROB_COLS, INTENSITY_COLS, DISASTERS_FOUND`

Reply:
For some weird reason I tried this yesterday like 10 times and it did not work. That is way I changed it. But I just tried and it worked.

Change made!